Oracle/cdm_transform_tests.sql

```diff
@@ -597,6 +597,38 @@ order by 2
 */
 
 
+/* "Low bar" test to verify that DISPENSING is not empty */
```

Comment:
In my container I didn't get any of the dispensing rows in the test_cases table.  I think this is because we 1/0 tests that fail before it (at least in my container).  I think these tests fail in production too.

Consider moving these new tests to the top of the file?